core-it-suite/src/test/java/org/apache/maven/it/MavenITmng0294MergeGlobalAndUserSettingsTest.java

```diff
@@ -53,16 +53,9 @@ public void testitMNG294()
         verifier.deleteDirectory( "target" );
         verifier.addCliOption( "--settings" );
         verifier.addCliOption( "user-settings.xml" );
-        if ( matchesVersionRange( "[2.1.0-M2,)" ) )
-        {
-            // dedicated CLI option only available since MNG-3914
-            verifier.addCliOption( "--global-settings" );
-            verifier.addCliOption( "global-settings.xml" );
-        }
-        else
-        {
-            verifier.setSystemProperty( "org.apache.maven.global-settings", "global-settings.xml" );
-        }
+        verifier.addCliOption( "--global-settings" );
+        verifier.addCliOption( "global-settings.xml" );
+
         if ( matchesVersionRange( "(3.2.5,)" ) )
         {
             verifier.getSystemProperties().put( "maven.multiModuleProjectDirectory", testDir.getAbsolutePath() );
```

Comment:
Can't this be simplified as well since we have now 3.9.0+?

Reply:
Even more `multiModuleProjectDirectory` is set by Verifier 1.8.0